rotkehlchen/exchanges/coinbase.py

```diff
@@ -285,49 +381,56 @@ def _api_query(
             options: dict[str, Any] | None = None,
             ignore_pagination: bool = False,
     ) -> list[Any]:
-        """Performs a coinbase API Query for endpoint
+        """
```

Comment:
Please undo this change as I asked before: https://github.com/rotki/rotki/pull/7903/files#r1593544499

Reply:
I lost this conversation due to issues with Rebase.

Reply:
Just a heads up on why you might not see the old commits: I attempted to cherry-pick my previous commits, but this process led to corruption in my fork when I tried to rebase. To resolve this and clean up the repository, I ended up losing those old commits.

Reply:
Hmmm this is not undone. You still are touching this docstring by just adding a newline and nothing else.